README.md

```diff
@@ -0,0 +1,55 @@
+# here-is-my-card
+
+## About
+Eutamin-C Sample Project<br>
+This is an android application using [Euphony library](https://github.com/euphony-io/euphony)<br>
+"here-is-my-card" makes it easy to **share business cards** with others.<br>
+With this application, you don't have to carry around your business card anymore!<br>
+
+## Features
+
+- Create your own business card using various and beautiful templates
+- Send your business card to others
+- Receive bussiness cards of someone you meet and save
+- All sending and receiving processes are carried out using [Euphony library](https://github.com/euphony-io/euphony)
+
+## How to use
+
+1. Select a business card template that you want.
+
+<img src = "https://user-images.githubusercontent.com/68395698/130160475-4242d19d-e795-454e-a782-242ef203ed60.gif" width =30% />
+
+2. Enter your information on your business card and press the plane button. Then the speaker turns on and the listener turns off.
+
+<img src = "https://user-images.githubusercontent.com/68395698/130109455-c5de67ed-545b-460c-ae74-9d4cbb307184.png" width =30% /> <img src = "https://user-images.githubusercontent.com/68395698/130109811-d8a684a5-b967-45cc-944f-64d2121840f0.png" width =30% /> <img src = "https://user-images.githubusercontent.com/68395698/130109759-e812a94b-3d68-4c27-8328-99e62f35280c.png" width =30% />
+
+3. The listener always works when you turn on the app. Press the lower right button and check business cards sent.  That's it!
```

Comment:
as far as I know, listeners don't always work.
listener works when SPEAK OFF & LISTEN ON.
@psh2849 ! right???
Let me know if I'm wrong!

Reply:
> as far as I know, listeners don't always work.
> listener works when SPEAK OFF & LISTEN ON.
> @psh2849 ! right???
> Let me know if I'm wrong!

When the app is run, the permission is checked.
And if the permission is allowed, the 'playListen' function is called.
So, It is correct that when you run the app, it automatically listens.

Reply:
That's right!  In 2, I wrote that "Then the speaker turns on and the listener turns off." I wrote this sentence "The listener always works when you turn on the app" to mean that there is no specific speaker turn off or turn on button. 
But I think this sentence was ambiguous!! I'll fix it !

Reply:
How about "The listener works except when the speaker turns on." ???

Reply:
![image](https://user-images.githubusercontent.com/66951780/130174126-7b358a42-e1a9-4bf5-99ca-095ea2fa6988.png)

Isn't it off here???

Reply:
@seoh02h That's Perfect!

Reply:
If the speaker and listener are turned on at the same time,
it listens to your own device.
So, When you run the app, turn on the listener, and turn off the speaker.
After you make a business card and clicked airplane picture, turn off the listener and turn on the speaker.  

As a result, the listener is not always turn on. but is temporarily turned on when the app is launched.

Reply:
@psh2849 Thanks for the detailed answer! 😄

Reply:
It was hard for me to receive fully correct information on my phone,,,,, So I used cardboard saved screenshot of @psh2849 !! Special Thanks!!